### DISCUSSION TITLE
GitHupb wrong session IP and authentication error 404

### DISCUSSION THREAD

**Discussion by Vitor-rs** [community/community#23492] [New to GitHub]
<p>Hello guys,<br>
I’m a newbie in IT and I’ve been trying to log in to my GitHub account, but with no avail. It keeps showing an error message/page saying that the page is not found. I can only connect into my account in inognito mode and I don’t know why. When I do it, I can get to the password and authentication settings and on the SESSIONS area, the IP that I am supposed to be using is wrong (from anather place). It is not my IP at all. So, can anyone tell me why the heck is this happening?</p>

**Comment by ettaboyle** [comment, marked as answer]
<p>:wave:t2: Hi <a class="mention" href="/vitorgit93">@VitorGit93</a> – First, welcome! That sounds frustrating and definitely not a great start to your IT experience.</p>
<p>The best place to go for answers to account-specific questions like this one is by opening a ticket on our <a href="https://support.github.com/contact">Support page</a>. Please open a ticket with them :slight_smile:</p>
<p>Unfortunately, we can’t help with account-specific questions on the public forum. We appreciate you being here and encourage you to check out the resources in the <a class="hashtag" href="/tag/getting-started-with-github-pages">#<span>getting-started-with-github-pages</span></a>  section</p>
